pkg/reconciler/trustedresources/verify_test.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trustedresources
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"go.uber.org/zap/zaptest"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	namespace = "tekton-pipelines"
+	password  = "hello"
+)
+
+var (
+	// tasks for testing
+	taskSpecTest = &v1beta1.TaskSpec{
+		Steps: []v1beta1.Step{{
+			Image: "ubuntu",
+			Name:  "echo",
+		}},
+	}
+)
+
+func init() {
+	os.Setenv("SYSTEM_NAMESPACE", namespace)
```

Comment:
I think the preferred way of doing this is is to import `knative.dev/pkg/testing`.

I'm also curious why you need this if we're not verifying any controller functionality yet 🤔

Reply:
This should be the piece of code I forgot to remove 🥲